lib/facebook/messenger/incoming/standby.rb

```diff
@@ -0,0 +1,63 @@
+module Facebook
+  module Messenger
+    module Incoming
+      #
+      # The Standby class represents an incoming Facebook Messenger when the handover-protocol is enabled.
+      # It's possible to receive:
+      #  - message_reads
+      #  - message_deliveries
+      #  - messages
+      #  - messaging_postbacks
+      #
+      # IMPORTANT: the postbak payload is received here only if CF has sent the original postback button
+      #
+      # @see https://developers.facebook.com/docs/messenger-platform/handover-protocol
+      # @see https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/standby
+      #
+      class Standby
+        attr_reader :messaging, :events
+
+        #
+        # Assign message to instance variable
+        #
+        # @param [Object] messaging Object of message.
+        #
+        def initialize(messaging)
+          @messaging = messaging
+          @events = build_events
+        end
+
+        #
+        # Function return Page ID from where we received the event.
+        #
+        # @return [String] Page ID
+        #
+        def id
+          messaging['id']
+        end
+
+        #
+        # Function return timestamp when message is sent.
+        #
+        # @return [Object] Message time sent.
+        #
+        def time
+          Time.at(messaging['time'] / 1000)
+        end
+
+        private
+
+        def build_events
+          messaging['standby'].map do |event|
+            key = (event.keys - %w[sender recipient timestamp]).first
```

Comment:
this should give us the missing key which will distinguish the different events - is there a better way to do so?

Reply:
Hm, I think that's ok for now.

But, what if FB adds more keys, and the one we need is not `.first` ? We'd want to always be sure we could handle it appropriately.